README.md

```diff
@@ -0,0 +1,33 @@
+# Dwellingly Frontend
+
+This serves as the Frontend of the Full Stack Application for [JOIN Organization's](https://joinpdx.org/) Dwellingly site being created per the FIGMA designs. This site will eventually aid in supporting both tenants
+
+
+## Prerequisite Technologies
+
+- [Node && npm](https://nodejs.org/en/download/)
+- [Git](https://gist.github.com/derhuerst/1b15ff4652a867391f03)
+
+
+### Getting Started
+
+Once necessary technologies are installed on your machine:
+
+From Terminal:
+
+- Navigate to your desired directory from the command line:
+    example:`cd Desktop/CodeForPDX/Dwellingly`
+- Clone Project - `git clone https://github.com/codeforpdx/dwellingly-app.git`
+- Create your own working branch:
+    example: `git checkout -b add-new-component`
+
+
+### Installing
```

Comment:
Let's also add a link to the dwellinglybackend project to tell newcomers where to look for the API project and its separate Readme

Reply:
YES! I had this on my list and completely forgot to do it before my PR yesterday! Thanks, @Chris-Boe!